client/asset/btc/electrum_client.go

```diff
@@ -895,10 +900,35 @@ func (ew *electrumWallet) pass() (pw string, unlocked bool) {
 	return ew.pw, ew.unlocked
 }
 
+func (ew *electrumWallet) testPass(pw []byte) error {
+	addr, err := ew.wallet.GetUnusedAddress(ew.ctx)
+	if err != nil {
+		return err
+	}
+	wifStr, err := ew.wallet.GetPrivateKeys(ew.ctx, string(pw), addr)
+	if err != nil {
+		if strings.Contains(strings.ToLower(err.Error()), "incorrect padding") {
+			return errors.New("incorrect password (no password required?)")
+		}
```

Comment:
Ah, yeah that's not at all clear.  I discovered that if the wallet was not password protected, but you provided a password, it would return the cryptic "incorrect padding" response.  I'm pretty sure I got "incorrect padding" in another case as well, so I didn't want to state definitively that it was "no password required", rather that's a possibility and it's definitely not the right password.